python/cudf/cudf/tests/test_dataframe_copy.py

```diff
@@ -171,7 +152,10 @@ def test_kernel_shallow_copy():
     gdf = DataFrame.from_pandas(pdf)
     cdf = gdf.copy(deep=False)
     sr = gdf["a"]
-    add_one[1, len(sr)](sr.to_cupy())
+    # Write a value in-place on the shallow copy.
```

Comment:
Make the newline spacing in this test match the other test.
```suggestion

    # Write a value in-place on the shallow copy.
```